sdcm/sct_config.py

```diff
@@ -1100,9 +1100,16 @@ class SCTConfiguration(dict):
 
         dict(name="new_version", env="SCT_NEW_VERSION", type=str,
              help="Assign new upgrade version, use it to upgrade to specific minor release. eg: 3.0.1"),
+
         dict(name="target_upgrade_version", env="SCT_TARGET_UPGRADE_VERSION", type=str,
              help="Assign target upgrade version, use for decide if the truncate entries test should be run. "
                   "This test should be performed in case the target upgrade version >= 3.1"),
+
+        dict(name="disable_raft", env="SCT_DISABLE_RAFT", type=boolean,
+             help="As for now, raft will be enable by default in all [upgrade] tests, so this flag will allow us"
+                  "to still run [upgrade] test without raft enabled (or disabling raft), so we will have better"
+                  "coverage"),
+
```

Comment:
I think that we should reverse it.
At the moment, the plan is not to enable raft on upgrades by default, but only who explicitly choose to do it.
So, our tests should behave the same.

The folder should be "upgrade-raft" and there test the combinations of raft enabled.

I know it's also a refactor of what we have, but it's better like that.